tests/helpers/fake-requests.js

```diff
@@ -104,8 +104,17 @@ function stubGetImageSets() {
   });
 }
 
+function stubDeleteImageSets() {
+  stubRequest('delete', '/imageSets', function(request){
```

Comment:
at @redyaffle nice work, thanks for the acceptance test, this is failing with the error you mentioned, `FakeServer received unhandled request for DELETE /imageSets/24` because the URL your are stubbing here is different than what the API is actually calling. you should be stubbing `/imageSets/24`